src/public/parts/shells/chat/public/src/templates/attachment_preview.html

```
@@ -1,12 +1,24 @@
+		${showSpeechRecognitionMenu ? `
+		<div class="dropdown dropdown-top dropdown-end">
+			<button type="button" tabindex="0" class="btn btn-xs btn-ghost attachment-menu-button" data-i18n="chat.attachment.buttons.more"></button>
+			<ul tabindex="0" class="dropdown-content menu menu-sm bg-base-100 rounded-box z-30 shadow border border-base-300 w-36 p-1">
+				${showDownloadButton ? `<li><button type="button" class="download-button" data-i18n="chat.attachment.buttons.download"></button></li>` : ''}
+				<li><button type="button" class="speech-recognition-button" data-i18n="chat.attachment.buttons.recognize"></button></li>
+				${showDeleteButton ? `<li><button type="button" class="delete-button" data-i18n="chat.attachment.buttons.delete"></button></li>` : ''}
+			</ul>
+		</div>
+		` : `
 		${showDownloadButton ? `
 		<button class="download-button btn btn-xs btn-primary ${showDeleteButton ? 'join-item' : ''}" data-i18n="chat.attachment.buttons.download"><img src="https://api.iconify.design/line-md/download.svg" data-i18n="chat.attachment.buttons.downloadIcon" /></button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)